Code/Samples/Asteroids/GameState.cpp

```diff
@@ -71,12 +69,10 @@ void AsteroidGameState::BeforeWorldUpdate()
 
 void AsteroidGameState::ConfigureInputActions()
 {
-#if EZ_ENABLED(EZ_PLATFORM_WINDOWS)
-  ezInputDeviceXBox360::GetDevice()->EnableVibration(0, true);
-  ezInputDeviceXBox360::GetDevice()->EnableVibration(1, true);
-  ezInputDeviceXBox360::GetDevice()->EnableVibration(2, true);
-  ezInputDeviceXBox360::GetDevice()->EnableVibration(3, true);
-#endif
+  ezControllerInput::GetDevice()->EnableVibration(0, true);

```

Comment:
Wouldn't you need to wrap all of these calls in a check? Unless you provide a fallback NULL implementation for controller devices.

Reply:
You are right. Will wrap.